sos/policies/package_managers/__init__.py

```diff
@@ -192,6 +194,24 @@ def all_files(self):
             self.files = files.splitlines()
         return self.files
 
+    def pkg_by_path(self, path):
+        """Given a path, return the package that owns that path.
+
+        :param path:    The filepath to check for package ownership
+        :type path:     ``str``
+
+        :returns:       The package name or 'unknown'
+        :rtype:         ``str``
+        """
+        if not self.query_path_command:
+            return 'unknown'
+        try:
+            cmd = f"{self.query_path_command} {path}"
+            pkg = shell_out(cmd, timeout=5, chroot=self.chroot)
+            return pkg.replace('\n', ',') or 'unknown'
```

Comment:
Why the replace? For a case when a file path is owned by multiple packages? As this generates `pids_to_packages.json` with unexpected commas:

        "1": {
            "path": "/usr/lib/systemd/systemd",
            "package": "systemd-239-51.el8.x86_64,"
        },

(or add there `.rstrip(',')` ?)

Reply:
Ack to stripping a trailing comma.